src/processes.py

```
@@ -23,8 +23,6 @@ def main():
     for process_name in get_process_list('/etc/igcollect/stat_per_process.cfg'):
         process_name = process_name.replace('\n', '')
         cpu, mem = get_process_data(process_name)
-        if not cpu or not mem:
-            continue
```

It will emit garbage which is better than silently ignoring the problem.